test/jdk/java/net/httpclient/http2/PushPromiseContinuation.java

```diff
@@ -61,17 +66,28 @@
 import java.util.function.BiPredicate;
 
 import static java.nio.charset.StandardCharsets.UTF_8;
-import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.assertEquals;
 
 public class PushPromiseContinuation {
 
     static HttpHeaders testHeaders;
     static HttpHeadersBuilder testHeadersBuilder;
+    static int continuationCount;
```

Comment:
Since these three static variables are set by one thread and read by another - they should all be volatile.